### PR TITLE
Resolve issue 49

### LINKED ISSUE #49: Add setNodeLink function for the datacatalog pages
PR #46 introduces the `setNodeLink` function for the ide.html page. We should also add the same functionality to the datacatalog.html page to navigate to the stac-browser endpoint

### PR DESCRIPTION
**Pull request by alexandercyu**
Added function to datacatalog.html to only show nodes with stac browser.
Resolves #49

### CODE REVIEW

File: templates/site/datacatalog.html [hunk 1]
```diff
@@ -23,8 +23,33 @@
...
+
+            function setNodeLink(name, data, node_id_suffix) {
+
+                data.services.forEach((service) => {
+                    console.log(service)
+                    console.log(service.name)
+
+                    if(service.name === "stac-browser")
+                    {
+                        if(service.keywords.includes("catalog")) {
```
Comment (mishaschwartz) [inline templates/site/datacatalog.html:37]: Why are we checking this too?

File: templates/site/datacatalog.html [hunk 2]
```diff
@@ -23,8 +23,33 @@
...
         <script>
             function nodeFilter([_name, data]) {
-                return data.services && data.services.some(service => service.keywords.includes("catalog"))
+                return data.services && data.services.some(service => service.name.includes("stac-browser"))
             }
+
+            function setNodeLink(name, data, node_id_suffix) {
+
+                data.services.forEach((service) => {
+                    console.log(service)
```
Comment (mishaschwartz) [inline templates/site/datacatalog.html:32]: remove debugging code

File: templates/site/datacatalog.html [hunk 3]
```diff
@@ -23,8 +23,33 @@
...
+                    console.log(service.name)
+
+                    if(service.name === "stac-browser")
+                    {
```
Comment (mishaschwartz) [inline templates/site/datacatalog.html:36]: please be consistent with spacing, indentations, etc.

File: templates/site/datacatalog.html [hunk 4]
```diff
@@ -23,8 +23,33 @@
     {% with %}
         <script>
             function nodeFilter([_name, data]) {
-                return data.services && data.services.some(service => service.keywords.includes("catalog"))
+                return data.services && data.services.some(service => service.name.includes("stac-browser"))
```
Comment (mishaschwartz) [inline templates/site/datacatalog.html:26]: includes will look for substrings. Please match the name exactly

**Review by mishaschwartz — CHANGES_REQUESTED** [review]

**Comment by mishaschwartz** [comment]
Please add "Resolves/Closing" keywords in the description so that #49 will be resolved when this is merged

**Comment by alexandercyu** [comment]
> Please add "Resolves/Closing" keywords in the description so that #49 will be resolved when this is merged

Keywords added.

File: templates/site/datacatalog.html (hunk 2 above)
Comment (alexandercyu) [reply templates/site/datacatalog.html:32]: Debugging code removed.

File: templates/site/datacatalog.html (hunk 1 above)
Comment (alexandercyu) [reply templates/site/datacatalog.html]: It was from the previous iteration.

Comment (alexandercyu) [reply templates/site/datacatalog.html]: Removed the condition checking for "catalog" in keywords

File: templates/site/datacatalog.html (hunk 4 above)
Comment (alexandercyu) [reply templates/site/datacatalog.html]: Changed condition to match name exactly

File: templates/site/datacatalog.html (hunk 3 above)
Comment (alexandercyu) [reply templates/site/datacatalog.html]: Indentation fixed

**Review by mishaschwartz — APPROVED** [review]

**Pull request merged**